src/steps.js

```diff
@@ -167,6 +167,6 @@ http://perkframework.com/v1/guides/getting-started-os-x.html
 
 
 
-`;
+` +  (current ? '' : 'Be Aware your perky-cli version is outdated, there may be differences with the docs');
```

Comment:
With ES6 string interpolation you could stick this right inside of the previous message instead of concatenating. Also there's a small typo (perky-cli) and we should notify them how to upgrade their perk-cli if they want to:

eg.
```js
return `
  
  
 Your new project has successfully been created in ${dirPath}
 
 You should run:
 
 	cd ${dirPath} && npm install
 
 While dependencies for your new perk app are installing you can check
 out more info on how to use all the great features of perk at:
 
 http://perkframework.com/v1/guides/getting-started-os-x.html
  
  
 ${current ? '' : 'Be Aware your perk-cli version is outdated, there may be differences with the docs. To upgrade run `npm install-g perk-cli`'}
 `;
```